doc/Accessibility.md

```diff
@@ -0,0 +1,81 @@
+# Accessibility
+
+## Commands
+
+To make sure that commands work correctly with a Screen Reader turned on, it's important that the command is attached to the control that will be focused by the screen reader or else the command will not work.
+
+For example this is not good:
```

Comment:
can we add a better formatting for this, just in case we want to add "Index" to this in the future?
for instance... 
## Important Notes
### Not Working Code
### Working Code